pkg/model/employees.go

```diff
@@ -24,31 +24,35 @@ type Employee struct {
 	BaseModel
 
 	// basic info
-	FullName                string
-	DisplayName             string
-	TeamEmail               string
-	PersonalEmail           string
-	Avatar                  string
-	PhoneNumber             string
-	Address                 string
-	MBTI                    string
-	Gender                  string
-	Horoscope               string
-	PassportPhotoFront      string
-	PassportPhotoBehind     string
-	IdentityCardPhotoFront  string
-	IdentityCardPhotoBehind string
-	DateOfBirth             *time.Time
+	FullName               string
+	DisplayName            string
+	TeamEmail              string
+	PersonalEmail          string
+	Avatar                 string
+	PhoneNumber            string
+	Address                string
+	MBTI                   string
+	Gender                 string
+	Horoscope              string
+	PassportPhotoFront     string
+	PassportPhotoBack      string
+	IdentityCardPhotoFront string
+	IdentityCardPhotoBack  string
+	DateOfBirth            *time.Time
 
 	// working info
 	WorkingStatus WorkingStatus
 	JoinedDate    *time.Time
 	LeftDate      *time.Time
+	ChapterID     UUID
+	SeniorityID   UUID
+	PositionID    UUID
+	LineManagerID UUID
 	AccountStatus AccountStatus
 
 	// social services
 	BasecampID             string
-	BasecampAttachableSGID string
+	BasecampAttachableSgid string
```

Comment:
should be `ID` (ref: https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
👍